File: 2026-02/src/day_19/boj_9366/Main.java
package day_19.boj_9366;

import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();
        StringTokenizer st;

        int t = Integer.parseInt(br.readLine());
        for (int tc = 1; tc <= t; tc++) {
            st = new StringTokenizer(br.readLine());
            int a = Integer.parseInt(st.nextToken());
            int b = Integer.parseInt(st.nextToken());
            int c = Integer.parseInt(st.nextToken());

            if (a >= b + c || b >= a + c || c >= a + b) {
                sb.append("Case #").append(tc).append(": invalid!\n");
            } else if (a == b && b == c) {
                sb.append("Case #").append(tc).append(": equilateral\n");
            } else if (a == b || b == c || c == a) {
                sb.append("Case #").append(tc).append(": isosceles\n");
            } else {
                sb.append("Case #").append(tc).append(": scalene\n");
            }
        }

        System.out.println(sb);
    }
}

File: 2026-02/src/day_19/boj_9366/Main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    ios::sync_with_stdio(0);
    cin.tie(0);

    int t;
    cin >> t;

    for (int tc = 1; tc <= t; tc++) {
        int a, b, c;
        cin >> a >> b >> c;

        if (a >= b + c || b >= a + c || c >= a + b) {
            cout << "Case #" << tc << ": invalid!\n";
        } else if (a == b && b == c) {
            cout << "Case #" << tc << ": equilateral\n";
        } else if (a == b || b == c || c == a) {
            cout << "Case #" << tc << ": isosceles\n";
        } else {
            cout << "Case #" << tc << ": scalene\n";
        }
    }
}
